orangecontrib/text/widgets/owconcordance.py

```diff
@@ -151,10 +151,14 @@ class OWConcordance(OWWidget):
     ]
     outputs = [('Selected Documents', Table, )]
 
+    settingsHandler = PerfectDomainContextHandler(
+        match_values=PerfectDomainContextHandler.MATCH_VALUES_ALL
+    )
+
     autocommit = Setting(True)
     context_width = Setting(5)
-    word = Setting("")
-    # TODO Set selection settings.
+    word = ContextSetting("b", exclude_metas=False)
```

Comment:
Well the widget is still WIP, I have not done the cleaning yet. Anyway thanks for the comments.